less/components/about-page.less

```diff
@@ -0,0 +1,55 @@
+.about-page {
+  header {
+    margin-bottom: 2.5rem;
+    margin-top: 2.5rem;
+    padding-left: 2.5rem;
+    padding-right: 2.5rem;
+    @media screen and (min-width: @screen-sm) {
+      padding-left: 0;
+    }
+  }
+  p {
+    max-width: 50rem;
```

Comment:
This is making the three sub titles in the three icons not be centered.

The text that are problems are 
- Start a conversation on Twitter
- Email us anytime
- Connect on the Discourse forum